packages/smooth_app/lib/pages/user_management/sign_up_page.dart

```diff
@@ -243,14 +255,18 @@ class _SignUpPageState extends State<SignUpPage> with TraceableClientMixin {
             ),
             const SizedBox(height: space),
             ListTile(
-              leading: Checkbox(
-                value: _foodProducer,
-                fillColor: MaterialStateProperty.resolveWith(getCheckBoxColor),
-                onChanged: (final bool? value) {
-                  if (value != null) {
-                    setState(() => _foodProducer = value);
-                  }
-                },
+              onTap: () {
+                setState(() => _foodProducer = !_foodProducer);
+              },
+              contentPadding: EdgeInsets.zero,
+              leading: IgnorePointer(
```

Comment:
I would use `Icon(Icons.check_box)` and `Icon(Icons.check_box_outlined_blank)` instead, but why not.

Reply:
Indeed, it would also be possible.
But this way, this ensure the UI will be consistent across all our screens